components/sections/Skills.js

```diff
@@ -22,75 +23,15 @@ const Skills = React.forwardRef(({ handleIconToggle }, ref) => {
       <section className="flex flex-col overflow-y-auto">
         <div ref={ref}></div>
         <section className="flex flex-col px-6 pt-6 pb-6 gap-y-6">
-          {/* Core */}
-          <IconSelector
-            handleIconToggle={handleIconToggle}
-            title={"Core"}
-            iconType={"core"}
-            iconData={iconData}
-          />{" "}
-          {/* Scripting */}
-          <IconSelector
-            handleIconToggle={handleIconToggle}
-            title={"Scripting"}
-            iconType={"scripting"}
-            iconData={iconData}
-          />
-          {/* Code Editors */}
-          <IconSelector
-            handleIconToggle={handleIconToggle}
-            title={"Code Editors"}
-            iconType={"editors"}
-            iconData={iconData}
-          />
-          {/* Frontend */}
-          <IconSelector
-            handleIconToggle={handleIconToggle}
-            title={"Frontend"}
-            iconType={"frontend"}
-            iconData={iconData}
-          />
-          {/* Backend and DB */}
-          <IconSelector
-            handleIconToggle={handleIconToggle}
-            title={"Backend and Database"}
```

Comment:
I just realised this doesn't work for this case, I will need to define both the key and label when defining the categories.